src/panels/config/logs/error-log-card.ts

```diff
@@ -470,6 +471,13 @@ class ErrorLogCard extends LitElement {
   }
 
   private _fetchLogsFunction = () => {
+    if (this._boot === 0) {
+      return fetchHassioLogs;
```

Comment:
Why not add this logic to the `fetchHassioLogsBoots` function? There we can strip off the  `boots/${boot}` if `boot === 0`?